canopy/design-system/yarn.lock

```diff
@@ -0,0 +1,11387 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
No autogenerated files should be checked into the repo (at least on master).

Reply:
resolved